### PR TITLE
[Merged by Bors] - Getting started guide

### PR DESCRIPTION
**Pull request by maltesander**
# Description

fixes https://github.com/stackabletech/kafka-operator/issues/447

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/getting_started/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+This guide will get you started with Apache Kafka using the Stackable Operator. It will guide you through the installation of the Operator and its dependencies, setting up your first Kafka instance and create, write to and read from a topic.
+
+== Prerequisites
+
+You will need:
+
+* a Kubernetes cluster
+* kubectl
+* Helm
```
Comment (fhennig) [inline docs/modules/getting_started/pages/index.adoc:11]:
```suggestion
* optional: Helm
```

If I saw it right, Helm is optional if you install with stackablectl

File: docs/modules/getting_started/pages/first_steps.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+----
+include::example$code/getting-started.sh[tag=kcat-cleanup-data]
+----
+
+You successfully created a Kafka cluster and produced and consumed data.
+
+== What's next
+
+Have a look at the xref:ROOT:usage.adoc[] page to find out more about the features of the Operator such as encryption and authentication or how to connect to OPA.
```
Comment (fhennig) [inline docs/modules/getting_started/pages/first_steps.adoc:121]:
```suggestion
Have a look at the xref:ROOT:usage.adoc[] page to find out more about the features of the Kafka Operator.
```

File: docs/modules/getting_started/examples/code/zookeeper.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: simple-zk
+spec:
+  version: 3.8.0-stackable0.7.1
+  servers:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
```
Comment (fhennig) [inline docs/modules/getting_started/examples/code/zookeeper.yaml:13]:
```suggestion
```

File: docs/modules/getting_started/examples/code/kafka.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  version: 3.2.0-stackable0.1.0
+  zookeeperConfigMapName: simple-kafka-znode
+  config:
+    tls: null
+  brokers:
+    config:
+      resources:
+        storage:
+          logDirs:
+            capacity: '2Gi'
```
Comment (fhennig) [inline docs/modules/getting_started/examples/code/kafka.yaml:16]: I suggest we leave out all this resource stuff to keep it simple and instead demo this in more detail in a different document.

**Review by fhennig — CHANGES_REQUESTED** [review]
I'll render it locally and test the script now, but this is what I have for now.

File: docs/modules/getting_started/examples/code/kafka.yaml (hunk 4 above)
Comment (maltesander) [reply docs/modules/getting_started/examples/code/kafka.yaml]: done

File: docs/modules/getting_started/pages/installation.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+After you have installed stackablectl, run the following command to install all operators necessary for Kafka:
+
+[source,bash]
+----
+include::example$code/getting-started.sh[tag=stackablectl-install-operators]
+----
+
+The tool will show
+
+[source,bash]
```
Comment (fhennig) [inline docs/modules/getting_started/pages/installation.adoc:26]:
```suggestion
[source]
```

**Review by fhennig — APPROVED** [review]

**Pull request closed without merging**